### ISSUE TITLE
Seems to be mainly not working in FVTT v10

### ISSUE THREAD

**Issue by steveculshaw** [lipefl/swade-tools#153]
Sadly, the module seems to be mainly not working in FVTT v10

E.g., the initial dialog for a melee weapon attack is shown with the attack and damage button, and the rolls are shown in chat

But it's not picking up the target details, so no calculation versus Parry, and subsequent damage versus Toughness 😭

Any plans on a update to FVTT v10 ?
... really hoping the answer is "due to release very soon" 😄

**Comment by lipefl** [comment]
Hi! Yes i intend to update to v10. I wouldn't say "soon", because i dont know how much work it'll take. And I haven't had the time to test yet. If you can help, please list me everything that's not working, it'll sure speed up the proccess.

Btw, how's savage worlds playing on v10?
Last I heard Savage Worlds official modules are "to be updated to v10". Is it working fine?

**Comment by steveculshaw** [comment]
Phew ... I'm very relieved that you plan to update to v10, as your module is essential to playing SWADE.

I'll try and collate the issues I'm finding.

I've only got the "core" SWADE premium module from Pinnacle, and upgraded to v10 as they'd released their update for v10. No Deadlands, Pathfinder, etc.
Looked to be working ok.

Again many thanks for a great module 👍

**Comment by samuelkeenan** [comment]
> Hi! Yes i intend to update to v10. I wouldn't say "soon", because i dont know how much work it'll take. And I haven't had the time to test yet. If you can help, please list me everything that's not working, it'll sure speed up the proccess.

The main thing I noticed that wasn't working is the automation surrounding targeting. You are able to open up cards and they seem to do the things they are supposed to (damage rolls damage, Fight skill rolls that skill, modifiers modify), but none of them actually target the enemy. So when I roll to shoot, I get that roll, but doesn't tell me if I hit/miss/raise. If I roll Damage, it doesn't tell me if I beat the damage threshold and doesn't display wounds/shaken and doesn't let me apply it. 

So basically, it appears like most of the targeted automation isn't functioning

The console suggests it's having an issue with Rollcontrol.findTargets and that targets is undefined. It seems to potentially be an issue with ChatMessages and how that has changed or with how the Data Model changed (but that's just speculation, I'm not well versed in JS or in module making).  

Let me know if you need anything looked at, the mod is pretty great and I'd love to be able to use it again.

**Comment by lipefl** [comment]
Hi! I released v1.10 compatible with v10. 
I had to do a lot of minor changes, so... it's still beta, but it's working now.
Please, test it out. If you find any other bugs, feel free to report.
